relay-metrics/src/protocol.rs

```diff
@@ -163,6 +163,90 @@ fn is_valid_name(name: &str) -> bool {
     false
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum SessionsKind<'a> {
+    Session,
+    User,
+    Error,
+    Other(&'a str),
+}
+
+impl<'a> Display for SessionsKind<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Session => write!(f, "session"),
+            Self::User => write!(f, "user"),
+            Self::Error => write!(f, "error"),
+            Self::Other(s) => write!(f, "{}", s),
+        }
+    }
+}
+
+impl<'a> From<&'a str> for SessionsKind<'a> {
+    fn from(value: &'a str) -> Self {
+        match value {
+            "session" => Self::Session,
+            "user" => Self::User,
+            "error" => Self::Error,
+            s => Self::Other(s),
+        }
+    }
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum TransactionsKind<'a> {
+    Duration,
+    User,
+    CountPerRootProject,
+    MeasurementsFramesFrozen,
```

Comment:
There are so many `measurements.`  variants, and there's also a place in the code which dynamically creates names that begin with `measurements.`. 

Perhaps we could create a `Measurements(String)` variant.